src/views/Download.vue

```diff
@@ -179,5 +217,14 @@ export default {
         white-space: nowrap;
       }
     }
+    
+    table {
+      border-spacing: 0 20px;
```

Comment:
Prefer the unit `rem` instead of `px` so it's relative to the root font size. Replace `20px` with `1rem` for example.